tst/testinstall/matrix.tst

```diff
@@ -0,0 +1,47 @@
+#
+gap> IsDiagonalMat(NullMat(3, 3));
+true
+gap> IsDiagonalMat(NullMat(1, 3));
+true
```

Comment:
Why do we want this to be this way?

Reply:
After consulting the manual again it is at least documented, including a reason.